test/xapi/track_test.rb

```diff
@@ -2,6 +2,16 @@
 require_relative '../../lib/xapi'
 
 class TrackTest < Minitest::Test
+  def assert_archive_contains(filenames, zip)
+    files = []
+    Zip::InputStream.open(zip) do |io|
+      while (entry = io.get_next_entry)
+        files << entry.name
+      end
+    end
+    assert_equal filenames.sort, files.sort
+  end
```

Comment:
> The arguments to the method are in the same order as they are in the method name

Most assert methods use the ordering: expected, actual. What about `assert_filenames_in_archive`? Too clunky?